conn.go

```diff
@@ -463,7 +473,7 @@ func (hc *halfConn) encrypt(b *block, explicitIVLen int) (bool, alert) {
 			payloadLen := len(b.data) - recordHeaderLen - explicitIVLen
```

Comment:
I would comment that, when this function is called and the connection is using 1.3,  `explicitIVLen` is guaranteed to be `0`.

Reply:
done